nwb_conversion_tools/baserecordingextractorinterface.py

```diff
@@ -42,20 +42,16 @@ def get_metadata_schema(self):
 
     def get_metadata(self):
         """Auto-fill as much of the metadata as possible. Must comply with metadata schema."""
-        out = super().get_metadata()
-        out['properties'].update(
+        metadata = super().get_metadata()
+        metadata.update(
             Ecephys=dict(
-                ElectricalSeries=dict(
-                    name='ElectricalSeries',
-                    description='raw acquired data'
-                ),
-                Device=dict(
-                    name='device',
-                    description='ecephys probe'
-                )
+                ElectricalSeries=dict(),
+                Device=[],
```

Comment:
why remove these? They seem like good defaults for a base recording extractor interface

Reply:
Because `NwbRecordingExtractor.write_recording()` is setup with similar defaults, but more robustly in the sense of interactions across `Electrodes` -> `ElectrodeGroups` -> `Device`. But  `NwbRecordingExtractor.write_recording()` can't fill with these defaults correctly unless it receives either `metadata=None`, or a metadata with each of these fields as empty.

Reply:
I see. I think we can handle that with options on deep_dict_update, but let's worry about that later.

Reply:
Agreed; there is probably a better way to set all these types of defaults but it will be easier to see and fix when all the other pieces currently in development are in place.